.dev.goreleaser.yml

```diff
@@ -0,0 +1,31 @@
+builds:
+  - main: ./cmd/notation
+    id: notation
+    binary: notation
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    ignore:
+      - goos: windows
+        goarch: arm64
+    ldflags:
+      - -s -w -X {{.ModulePath}}/internal/version.Version={{.Version}} -X {{.ModulePath}}/internal/version.BuildMetadata=
+archives:
+  - format: tar.gz
+    format_overrides:
+      - goos: windows
+        format: zip
+    files:
+      - LICENSE
+release:
+  prerelease: auto
+  header: |
+    ## Notation Weekly Release ({{ .Date }})
+
+    Welcome to this weekly release!
```

Comment:
@yizha1 @FeynmanZhou Do you have suggestion on the release note template?

Reply:
It's better to change "weekly release" to "Weekly Dev Build".

Reply:
Done.